internal/drivers/ankabuild/driver.go

```diff
@@ -0,0 +1,177 @@
+package ankabuild
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/drone-runners/drone-runner-aws/internal/drivers"
+	"github.com/drone-runners/drone-runner-aws/internal/lehelper"
+	"github.com/drone-runners/drone-runner-aws/types"
+	"github.com/drone/runner-go/logger"
+
+	"github.com/sirupsen/logrus"
+)
+
+type config struct {
+	username    string
+	password    string
+	rootDir     string
+	vmID        string
+	userData    string
+	nodeID      string
+	registryURL string
+	authToken   string
+	tag         string
+	ankaClient  Client
+}
+
+const (
+	startupScriptCondition = 0
+	scriptFailHandler      = 2
+	scriptTimeout          = 200
+	ankaRunning            = "Running"
+)
+
+func New(opts ...Option) (drivers.Driver, error) {
+	c := new(config)
+	for _, opt := range opts {
+		opt(c)
+	}
+	if c.ankaClient == nil {
+		c.ankaClient = NewClient(c.registryURL, c.authToken)
+	}
+	return c, nil
+}
+
+func (c *config) Create(ctx context.Context, opts *types.InstanceCreateOpts) (instance *types.Instance, err error) {
+	startTime := time.Now()
+	uData := base64.StdEncoding.EncodeToString([]byte(lehelper.GenerateUserdata(c.userData, opts)))
+	machineName := fmt.Sprintf(opts.RunnerName+"-"+"-%d", startTime.Unix())
+
+	logr := logger.FromContext(ctx).
+		WithField("cloud", types.AnkaBuild).
+		WithField("name", machineName).
+		WithField("pool", opts.PoolName)
+
+	logr.Info("Starting Anka Build Setup")
```

Comment:
https://github.com/golang/lint/issues/41
first letter of the log message should be lower case. yes we are in-consistent but i have fixed the other providers in a separate pr.

Reply:
done